### PR TITLE
Add dependencies in meta file

### PR DESCRIPTION
**Pull request by mkotsbak**
---
name: Pull request
about: Describe the proposed change

---

**Describe the change**
A clear and concise description of what the pull request is.

**Testing**
In case a feature was added, how were tests performed?

### CODE REVIEW

**Comment by robertdebock** [comment]
Hm, that's a very good one. I [typically use](https://robertdebock.nl/my-manifesto.html) no dependencies (see under "Dependecies"), but for reboot this may be a very good solution.

Give me a moment to test and think about it.

**Comment by robertdebock** [comment]
If I read the [documentation on dependencies](https://docs.ansible.com/ansible/latest/user_guide/playbooks_reuse_roles.html#role-dependencies) I understand that these dependencies are:
- Downloaded (good option)
- Executed before (that would not work here)

I'll do some testing, see if the dependencies would work here.

File: meta/main.yml [hunk 1]
```diff
@@ -41,4 +41,6 @@ galaxy_info:
     - rhel
     - ubuntu
 
-dependencies: []
+dependencies:
+  - robertdebock.bootstrap
```
Comment (robertdebock) [inline meta/main.yml:45]: The bootstrap  role is not a "hard requirement". Systems can also be prepared by something else, as long as they have python and sudo, Ansible will be able to manage the machine.

I've added the bootstrap dependency in `molecule/*/playbook.yml` to prepare a container. Most are pretty dressed down, so do not have python.

Comment (mkotsbak) [reply meta/main.yml:45]: So add only reboot then?

**Comment by robertdebock** [comment]
Yes, only reboot is a hard dependency.

And as you can see GitHub has decided that "DCO" is required. I've changed the code for you.

Thanks @mkotsbak, your contribution made me change differently about dependencies!

**Pull request closed without merging**